site/installation/migration.md

```diff
@@ -327,8 +327,8 @@ kpt `v0.39`) to `v1` version(compatible with kpt `v1.0`).
 [openapi validations]: https://googlecontainertools.github.io/kpt/guides/producer/setters/#openapi-validations
 [required setters]: https://googlecontainertools.github.io/kpt/guides/producer/setters/#required-setters
 [auto-setters]: https://googlecontainertools.github.io/kpt/concepts/setters/#auto-setters
-[migrating inventory objects]: https://googlecontainertools.github.io/kpt/reference/cli/live/alpha/
```

Comment:
change the wording for this link, it's not a "detailed Guide".

Reply:
updated.